app-admin/sshguard/sshguard-99999.ebuild

```diff
@@ -11,12 +11,14 @@ EGIT_REPO_URI="https://bitbucket.org/${PN}/${PN}"
 LICENSE="BSD"
 SLOT="0"
 KEYWORDS=""
+IUSE="systemd"
```

Comment:
This is not what we use the flag for. Remove it and the PR should be fine.

Reply:
Done. Thank you @mgorny